ci/autoscaler/scripts/cleanup-autoscaler.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 set -euo pipefail
 system_domain="${SYSTEM_DOMAIN:-autoscaler.ci.cloudfoundry.org}"
-service_name="${SERVICE_NAME:-autoscaler}"
```

Comment:
this should stay for backward compat.

Reply:
I think these cleanup scripts are only used by our CI. so if that's the case we can clean up whatever is breaking because of this manually if it fails.